WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -1044,6 +1044,7 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
             .mentions: FeatureFlag.gutenbergMentions.enabled && SuggestionService.shared.shouldShowSuggestions(for: post.blog),
             .xposts: FeatureFlag.gutenbergXposts.enabled && SiteSuggestionService.shared.shouldShowSuggestions(for: post.blog),
             .contactInfoBlock: post.blog.supports(.contactInfo) && FeatureFlag.contactInfo.enabled,
+            .layoutGridBlock: FeatureFlag.layoutGrid.enabled,
```

Comment:
I believe this meant to also call into `post.blog.supports(.layoutGrid)`

Reply:
Good catch this is fixed now.